exchange-proxy/deployment.md

```diff
@@ -0,0 +1,66 @@
+# Exchange Proxy Deployment
+
+This document outlines procedures for deploying and upgrading the Exchange Proxy.
+
+## Overview
+
+The Exchange Proxy is composed of several independently deployed contracts that broadly fit into these categories:
+
+- [Exchange Proxy](./exchange-proxy) (`ZeroEx`)
+    - This is the Exchange Proxy instance itself. It serves as the entry point for all transactions, which it will reroute to the appropriate feature contract.
+- [Features](./features)
+    - These contracts contain the implementation code for features that will be registered to the Exchange Proxy. The Exchange Proxy will ultimately `delegatecall` into these contracts.
+- Migration contracts
+    - These are one-shot contracts that aid in bootstrapping/configuring the Exchange Proxy during deployment. These contracts will typically self-destruct after use.
+- [Transformers](./features/transform-erc20#transformers)
+    - These are the composable transformations used by the `TransformERC20` feature.
+- [`TransformerDeployer`](https://github.com/0xProject/0x-monorepo/blob/development/contracts/zero-ex/contracts/src/external/TransformerDeployer.sol)
+    - This is the contract that will be used to deploy the individual transformer contracts. Transformers [must](./features/transform-erc20#locking-down-transformers) be deployed by a designated address.
+
+This diagram loosely illustrates the direct dependencies of each contract in the system.
+
+![deployment dependency graph](./img/ep-deployment-deps.png)
+
+Once these contracts are deployed, they will need to be wired together using the [`FullMigration`](https://github.com/0xProject/0x-monorepo/blob/development/contracts/zero-ex/contracts/src/migrations/FullMigration.sol) contract.
+
+### Administration
+
+The Exchange Proxy exposes a few admin-level functions, which are only accessible to its owner. In a production deployment, the owner would be set to the [`ZeroExGovernor`](https://github.com/0xProject/0x-protocol-specification/blob/master/v3/zero-ex-governor.md) multisig contract.
+
+## Deployment Steps
+
+### 1. Deploy the contracts
+The following contracts need to be deployed individually. With the exception of transformers, any address may deploy these contracts.
+- `FullMigration`
+- `ZeroEx`
+    - Passing `FullMigration.getBootstrapper()` to the constructor.
+- Features:
+    - `Ownable`
+    - `SimpleFunctionRegistry`
+    - `SignatureValidator`
+    - `TokenSpender`
+    - `TransformERC20`
+    - `MetaTransactions`
+- `TransformerDeployer`
+- Transformers (deployed through `TransformerDeployer`)
+
+### 2. Wire it all together
```

Comment:
we should link to the `migration.ts` file in `@0x/contracts-zero-ex` somewhere in here